src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            if (i < 3) {
```

Comment:
```suggestion
            if (i < figures.length/2) {
```

такая запись будет более масштабируемой (если нам нужно будет поменять коллличество генерируемых фигер, тонужно будет изменить только 1 число - размер массива)